<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QDialog" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonOpenBci">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;</string>
   </property>
   <property name="text">
    <string>OpenBci</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonMode">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEyeX">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;</string>
   </property>
   <property name="text">
    <string>EyeX</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
